service-maintenance/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM node:14.13.1-alpine3.12 as asset-env
+
+WORKDIR /app
+
+COPY web/assets web/assets
+COPY webpack.config.js .
```

Comment:
We're generally trying to remove `webpack` due to the dependency churn it creates. `eslint` is faster and has fewer dependencies, so is a good fit especially for smaller services that won't have the dev power to keep up with all the churn.

Reply:
I'll be attempting this in a later ticket https://opgtransform.atlassian.net/browse/UML-2383